apps/antalmanac/src/components/Calendar/CalendarRoot.tsx

```diff
@@ -82,7 +82,7 @@ export default function ScheduleCalendar(props: ScheduleCalendarProps) {
     const { isMilitaryTime } = useTimeFormatStore();
     const { hoveredCourseEvents } = useHoveredStore();
 
-    const getEventsForCalendar = () => {
+    const getEventsForCalendar = (): CourseEvent[] => {
         return showFinalsSchedule
             ? finalsEventsInCalendar
             : hoveredCourseEvents
```

Comment:
Type '(CourseEvent | CustomEvent)[]' is not assignable to type '(CourseEvent | CustomEvent<any>)[]'.
  Type 'CourseEvent | CustomEvent' is not assignable to type 'CourseEvent | CustomEvent<any>'.
    Type 'CustomEvent' is not assignable to type 'CourseEvent | CustomEvent<any>'.
      Type 'CustomEvent' is missing the following properties from type 'CourseEvent': locations, showLocationInfo, finalExam, courseTitle, and 4 more.ts(2322)
      
TS is screaming bloody murder here, might be best to untype it.

Reply:
`CalendarEvent[]` should also be okay if you still want to give it a type since `CalendarEvent = CourseEvent | CustomEvent`